django/apps/blue_management/blue_mgnt/views/views.py

```diff
@@ -265,6 +265,27 @@ def set_api_version(api):
         api.update_enterprise_settings(dict(api_version=version))
 
 
+def sanitize_redirect(url):
+    parsed = urlparse.urlparse(url)
+    path = os.path.normpath(
+        parsed.path.strip()
+    )
+    # Change single dot to slash in case it's just a plain domain
+    if path == '.':
+        path = '/'
+    without_netloc = urlparse.ParseResult(
+        scheme=None, netloc=None,
+        path=path,
+        params=parsed.params,
+        query=parsed.query,
+        fragment=parsed.fragment
```

Comment:
I would not even add params, query or fragment here. I'm not a big fan of this comparison, but if @bdzim likes it, I'm ok.

Reply:
Yes, it's feels odd, it work if you leave the dot but it would not look great in the logs IMHO. I've just added @bdzim